docs/proposals/EntGAN.md

```diff
@@ -0,0 +1,33 @@
+# Integrate GAN and Self-taught Learning into Sedna Lifelong Learning to Handle Unknown Tasks
+
+## Motivation
+
+In the process of Sedna lifelong learning, there would be a chance to confront unknown tasks, whose data are always heterogeneous small sample. Generate Adversarial Networks(GAN) is the start-of-art generative model and GAN can generate fake data according to the distribution of the real data. Naturally, we try to utilize GAN to handle small sample problem. Self-taught learning is an approach to improve classfication performance using sparse coding to construct higher-level features with the unlabeled data. Hence, we combine GAN and self-taught learning to help Sedna lifelong learning handle unknown tasks.
+
+### Goals
+
+* Handle unknown tasks
+* Implement of a lightweight GAN to solve small sample problem
+* Utilize self-taught learning to solve heterogeneous problem
+
+## Proposal
+We focus on the process of handling unknown tasks.     
+
+The overview is as follows:
+
+![](images/EntGAN%20overview.png)
+
+The process is illustrated as below:    
+1. GAN exploits the unknown task sample to generate more fake sample. 
+2. Self-taught learning unit utilize the fake sample and orginal unknown task sample and its label to train a classifier.
+3. A well trained classifier is output.
+
```

Comment:
The architecture is needed for the proposal. A critical weak point of this proposal is that the proposed architecture has not yet considered the integration of Sedna or Ianvs.
 Especially what is the relation between the proposed modules and Sedna / Ianvs modules?

Member
@[MooreZheng](https://github.com/MooreZheng) MooreZheng [on 8 Aug](https://github.com/kubeedge/sedna/pull/337#discussion_r940173535)
In the new version, we see an integration, but there are still concerns about motivation.
See: https://github.com/kubeedge/sedna/pull/337#discussion_r940166481